### PR TITLE
ENT-427: Update actOnOffer flow to route to WOGS

### PR DESCRIPTION
**Pull request by DuriDuri**

### CODE REVIEW

File: lib/entities/job.js [hunk 1]
```diff
@@ -50,45 +50,25 @@ function addBillingDocument(client, id, docType) {
...
-
-      // Patch the existing appointment if one exists.
-      if (appointment) {
-        return client.doAuthenticatedRequest('PATCH', `${endpoints.APPOINTMENTS}/${appointment.id}`, appointmentBody, options);
-      }
+    const appointmentBody = {
+      appointment_duration: body.appointment_duration,
+      appointment_time: body.appointment_time,
+      user_id: body.user_id,
+    };
```
Comment (michael-pr) [inline lib/entities/job.js]: Why not just pass the body? It looks like it has the exact same params

File: lib/entities/job.js [hunk 2]
```diff
@@ -50,45 +50,25 @@ function addBillingDocument(client, id, docType) {
...
 
-      // Otherwise, create a new appointment.
-      return client.doAuthenticatedRequest('POST', endpoints.APPOINTMENTS, appointmentBody, options);
-    })
-    .then(() => {
-      return { status: 'scheduled' };
-    });
+    return client.doAuthenticatedRequest('POST', `${endpoints.WORK_ORDER_GOVERNANCE_SERVICE}/${id}/accept`, { appointment: appointmentBody }, options);
```
Comment (michael-pr) [inline lib/entities/job.js]: You return a custom response down below, why not here?

File: lib/entities/job.js (hunk 1 above)
Comment (DuriDuri) [inline lib/entities/job.js]: I have to aggregate it all into an `appointment:{}` object. Handling username here also means we dont need to change anything in Work

File: lib/entities/job.js (hunk 2 above)
Comment (DuriDuri) [inline lib/entities/job.js]: Good point, making the update now

**Pull request merged**